src/main/java/de/tum/in/www1/artemis/service/plagiarism/ContinuousPlagiarismControlResultsService.java

```diff
@@ -0,0 +1,102 @@
+package de.tum.in.www1.artemis.service.plagiarism;
+
+import static java.util.function.Predicate.isEqual;
+import static java.util.stream.Collectors.toUnmodifiableSet;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Stream;
+
+import org.jvnet.hk2.annotations.Service;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import de.tum.in.www1.artemis.domain.*;
+import de.tum.in.www1.artemis.domain.enumeration.AssessmentType;
+import de.tum.in.www1.artemis.domain.enumeration.FeedbackType;
+import de.tum.in.www1.artemis.domain.enumeration.Visibility;
+import de.tum.in.www1.artemis.domain.participation.StudentParticipation;
+import de.tum.in.www1.artemis.domain.plagiarism.PlagiarismResult;
+import de.tum.in.www1.artemis.repository.ResultRepository;
+import de.tum.in.www1.artemis.repository.SubmissionRepository;
+
+/**
+ * Manages results of checks executed during continuous plagiarism control.
+ */
+@Service
+@Component
+class ContinuousPlagiarismControlResultsService {
+
+    private static final Logger log = LoggerFactory.getLogger(ContinuousPlagiarismControlResultsService.class);
+
+    private static final String PLAGIARISM_RESULT_FEEDBACK_EN = "Suspicion of plagiarism! Score reduced to 0.";
```

Comment:
Please do not hardcode any language dependent String into the server code. Language dependent information generation shall solely be handled by the client.

Reply:
This is impossible to do on the client side, as feedback text is saved to the database. It is the same when tutor adds it manually. This text belongs to the feedback and not to interface or copy writing of the application

Reply:
Solution: Add a issue for it.

Reply:
You can take a look at static code analysis and submission policies. They also have this problem for feedbacks and manage to keep out translations from the code.
See e.g. `STATIC_CODE_ANALYSIS_FEEDBACK_IDENTIFIER` in feedback.model.ts

Reply:
Thanks @JohannesStoehr 

I've added the indentifier, improved copy writing and extracted related logic to separate methods

Reply:
I've added an issue for handling translations on the server side: #6913